File: app/src/main/java/org/take365/Helpers/DateHelpers.java
package org.take365.Helpers;

/**
 * Created by divere on 02/11/2016.
 */

public class DateHelpers {

    private static String[] monthNames = new String[]{
            "Январь",
            "Февраль",
            "Март",
            "Апрель",
            "Май",
            "Июнь",
            "Июль",
            "Август",
            "Сентябрь",
            "Октябрь",
            "Ноябрь",
            "Декабрь"
    };

    public static String getMonthDate(int num) {
        return monthNames[num];
    }

}

File: app/src/main/java/org/take365/helpers/DateHelpers.kt
package org.take365.helpers

/**
 * Created by divere on 02/11/2016.
 */

object DateHelpers {

    private val monthNames = arrayOf("Январь", "Февраль", "Март", "Апрель", "Май", "Июнь", "Июль", "Август", "Сентябрь", "Октябрь", "Ноябрь", "Декабрь")

    fun getMonthDate(num: Int): String {
        return monthNames[num]
    }
}
